Welcome.md

```diff
@@ -44,6 +47,33 @@ start using or contributing to our code.
     - [Prevent wrapping within psql (optional)](#prevent-wrapping-within-psql-optional)
 
 
+## Open source philosophy
+The Seattle Flu Study practices open science.
+This means that our code (with very few, notable exceptions) is all open source.
+Even though our work can be highly specific and not always applicable to the public domain, we use a "public by default" rather than a "private by default" approach to our developer material.
+We only privatize repositories or documentation that contain sensitive data or whose release could pose a security risk to the study or its members.
+Here is a quick guide to what's safe to share and what's not.
+
+### Safe to share publicly
+* Documentation that is Seattle Flu specific but not sensitive
+* Links to Slack conversations
+* Links to Trello cards
+* Links to Metabase (or other apps at backoffice.seattleflu.org)
+* First names of study contacts
+
+
+### Don't share publicly
+* Personally identifiable information of study participants are employees
```

Comment:
replace "are" with "or"

Reply:
Oops! Meant to push that in the fixup commit, but I missed it. I'll push it soon then rebase before merging.